lib/gh/download_file_from_release.go

```diff
@@ -31,11 +34,21 @@ var (
 	ErrFatalError            = errors.New("fatal error using github")
 )
 
+type File struct {
```

Comment:
The identifier `File` seems very generic. Maybe `VersionedFile` or `ReleasedFile` would be more specific?